wwwroot/css/lombiq-privacy-consent-banner.css.map

```diff
@@ -1 +1 @@
-{"version":3,"sources":["lombiq-privacy-consent-banner.scss","lombiq-privacy-consent-banner.css"],"names":[],"mappings":"AAAA;EACI,aAAA;EACA,eAAA;EACA,MAAA;EACA,QAAA;EACA,aAAA;EACA,gBAAA;ACCJ","file":"lombiq-privacy-consent-banner.css","sourcesContent":[".privacy-consent-banner {\r\n    z-index: 1500;\r\n    position: fixed;\r\n    top: 0;\r\n    right: 0;\r\n    margin-top: 0;\r\n    max-width: 500px;\r\n}\r\n",".privacy-consent-banner {\r\n  z-index: 1500;\r\n  position: fixed;\r\n  top: 0;\r\n  right: 0;\r\n  margin-top: 0;\r\n  max-width: 500px;\r\n}"]}
```

Comment:
I'm wondering if it's OK that the source maps will always only contain LFs - see my question [here in GE](https://github.com/Lombiq/Gulp-Extensions/pull/35#discussion_r856611395).

Reply:
It's fine. The source maps contain a copy of the source they reference (it is this copy we edit with `normalizeSourceMap` to make LF only) and the encoded mapping information in the map file uses line numbers to address the mapping, so this can't cause any desynchronicity. ([See the source map specification.](https://sourcemaps.info/spec.html#:~:text=each%20group%20representing%20a%20line%20in%20the%20generated%20file))

Reply:
Thanks for the clarification.

Do I understand correctly, that you've introduced this change so that generating the source map file will always lead to the same result regardless of the platform you're working on? Or was there a different reason?

Reply:
Yes, that's the reason. If I didn't do it, it would serialize the native line endings in the contents string, so `\r\n` on Windows and `\n` everywhere else. The repo on github will be LF only, so the serialized code in the map file should be too.

Reply:
Makes sense. I agree.